src/chains/ethereum/ethereum/tests/provider.test.ts

```diff
@@ -30,6 +30,185 @@ describe("provider", () => {
         });
       });
     });
+
+    it("uses the default 'clock' time for the `timestampIncrement` option", async () => {
+      const provider = await getProvider();
+      const options = provider.getOptions();
+      assert.strictEqual(options.miner.timestampIncrement, "clock");
+
+      const timeBeforeMiningBlock = Math.floor(Date.now() / 1000);
+      await provider.request({
+        method: "evm_mine",
+        params: []
+      });
+      const timeAfterMiningBlock = Math.floor(Date.now() / 1000);
+      const block = await provider.request({
+        method: "eth_getBlockByNumber",
+        params: ["latest", false]
+      });
+      // the `block.timestamp` can be the same as `timeBeforeMiningBlock` and/or
+      // `timeAfterMiningBlock` because the precision of `block.timestamp` is 1
+      // second (floored), and mining happens much quicker than 1 second.
+      assert(
+        parseInt(block.timestamp) >= timeBeforeMiningBlock,
+        `block wasn't mined at the right time, should have been on or after ${timeBeforeMiningBlock}, was ${parseInt(
+          block.timestamp
+        )}`
+      );
+      assert(
+        parseInt(block.timestamp) <= timeAfterMiningBlock,
+        `block wasn't mined at the right time, should have been on or before ${timeAfterMiningBlock}, was ${parseInt(
+          block.timestamp
+        )}`
+      );
+      await provider.disconnect();
+    });
+
+    it("uses the timestampIncrement option", async () => {
+      const time = new Date("2019-01-01T00:00:00.000Z");
+      const timestampIncrement = 5;
+      const provider = await getProvider({
+        chain: { time },
+        miner: { timestampIncrement }
+      });
+      await provider.request({
+        method: "evm_mine",
+        params: []
+      });
+      const block = await provider.request({
+        method: "eth_getBlockByNumber",
+        params: ["latest", false]
+      });
+      assert.strictEqual(
+        parseInt(block.timestamp),
+        Math.floor(+time / 1000) + timestampIncrement
+      );
+      await provider.disconnect();
+    });
+
+    it("uses time adjustment after evm_setTime when timestampIncrement is used", async () => {
```

Comment:
```suggestion
    it("uses time adjustment after `evm_setTime` when `timestampIncrement` is used", async () => {
```
Just to be consistent with the other tests